.github/actions/spelling/excludes.txt

```diff
@@ -99,3 +99,4 @@ ignore$
 ^\Qsrc/go/nerdctl-stub/nerdctl_commands_generated.go\E$
 ^\Q.github/workflows/config/.golangci.yaml\E$
 ^\Qsrc/go/guestagent/.golangci.yml\E$
+^\Qsrc/go/networking/.golangci.yml\E$
```

Comment:
Perhaps we should just ignore `/\.golangci\.yml\E$` instead of specific ones?

Reply:
I think we will remove these 2 files again and fold them into the global one.